src/lib/grad.jl

```diff
@@ -10,6 +16,27 @@ macro nograd(ex)
   return blk
 end
 
+"""
+    nograd(f)
+
+Calls `f()` without accumulating gradients.
+
+    julia> A = randn(5);
```

Comment:
can you give an example not mutating globals (it doesn't feel very julian). Maybe just place A within f(x)?

Reply:
Fixed.